routing/blinding.go

```diff
@@ -239,6 +271,12 @@ func (s *BlindedPaymentPathSet) ToRouteHints() (RouteHints, error) {
 	return hints, nil
 }
 
+// isBlindedRouteNUMSTargetKey returns true if the given public key is the
+// NUMS key used as a target for blinded path final hops.
+func isBlindedRouteNUMSTargetKey(pk []byte) bool {
+	return bytes.Equal(pk, BlindedPathNUMSKey.SerializeCompressed())
```

Comment:
to improve efficiency, can store `BlindedPathNUMSKey.SerializeCompressed()` as a global